src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,21 @@
+.logo {
+  display: flex;
+
+  &__image {
+    height: 46px;
+
+    @include onTablet {
+      height: 65px;
+    }
+
+    &--menu {
```

Comment:
The &--menu modifier should be a separate element under the .logo block, not a nested element under .logo__image.